### ISSUE TITLE
[Bug Report] New v-tab does not scroll into view with show-arrows

### ISSUE THREAD

**Issue by pbastowski** [vuetifyjs/vuetify#5398]
When I add more tabs that can fit in the width of the tabs bar, and the scroll arrows show, the first new tab does not scroll into view even though I set the v-model to it's index.

### Versions and Environment
Vuetify: 1.3.1
Vue: 2.5.17
Browsers: 69.0.3497.100
OS: MacOS High Sierra 10.13.6

### Reproduction Link

Here is a link to a repro: https://codepen.io/pbastowski/pen/pxOONM?editors=1100

### Steps to reproduce

If you keep clicking the ADD button at the end of the tabs bar you will notice that tab 5 is hidden, see image below.

![image](https://user-images.githubusercontent.com/5649024/47365446-dd131f80-d6db-11e8-8cba-e7984f4ca817.png)

If you add another tab, tab 6 on my screen, then both tabs 5 and 6 are fully revealed.

![image](https://user-images.githubusercontent.com/5649024/47365513-0764dd00-d6dc-11e8-93df-4ba4a6baf67a.png)

I have a workaround for this right now, see the code below. `tabs` is an array, which I use to render each `v-tab`. When the array's length increases I scroll the new tab into view using `scrollIntoView()`, but only after 700ms have passed, otherwise it does not work (it could be something to do with the animation, I guess).

```js
    watch: {
        /*
            Fix for the hidden new tab problem. 
            Always scroll the last added tab into view
         */
        'tabs.length'(nv, ov) {
            if (!this.$refs.tabs) return  // $refs doe not exist until mounted
            setTimeout(() => {
                if (nv === ov + 1)
                    try {
                        this.$refs.tabs.$el
                            .querySelector('.s-tab:last-child')
                            .scrollIntoView(false)
                    } catch (e) {
                        // silently ignore
                    }
            }, 700)
        }
    },
```

### Expected Behavior

The new tab scrolls into view automatically when I set the `v-model` on `v-tabs` to it's index, even when it's the first new tab that does not fit into the available width of the tabs bar.

### Actual Behavior

The newly added tab remains hidden. Adding another new tab, scrolls both new tabs into view, see images above.

**Comment by erapoport** [comment]
I see the same problem, not just with new tabs, but anytime the tab index is changed programatically.

This looks like a similar report - https://github.com/vuetifyjs/vuetify/issues/3699

**Comment by johnleider** [comment]
This is due to the width calculation that happens to before overflow is determined. Once overflow is determined, it updates the template to have extra margin for the arrows which causes the scrollOffset to be off. I marked this as v1.3 but I might move it back further.

**Comment by johnleider** [comment]
Fixed in f22d754
